cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -0,0 +1,23 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+===============================
+Transaction Manager Maintenance
+===============================
+
+.. highlight:: console
+
+Transactions are marked as invalid when the changes made during the transaction cannot be rolled back. Transaction
+Manager has to store all invalid transactions so that they can be excluded from all future reads. This invalid 
```

Comment:
How about: 

The Transaction Service keeps track of all invalid transactions to exclude their writes from all future reads. Over time this invalid list can grow and lead to performance degradation. To avoid that, you can prune the invalid list after the data of invalid transactions has been removed---which happens during major HBase compactions. 

After a major...